english/watch-movie/Up.md

```diff
@@ -5,5 +5,7 @@ Turns out it loves Agarn!
 You then change the channel to meet the press
 Marshmallow gets even more fascinated
 
-You change the channel to Casablanca, 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.